modules/cnf-installing-numa-resources-operator-cli.adoc

```diff
@@ -94,8 +94,8 @@ $ oc get csv -n openshift-numaresources
 +
 .Example output
 
-[source,terminal]
+[source,terminal,subs="attributes+"]
 ----
 NAME                            DISPLAY                      VERSION   REPLACES   PHASE
-numaresources-operator.v4.10.0  NUMA Resources Operator      4.10.0               Succeeded
+numaresources-operator.v{product-version}.2  NUMA Resources Operator      4.{product-version}.2               Succeeded
```

Comment:
To clarify, you added the zstream (.2)just to resemble a real build right?

Reply:
Yes I ran this against a real build. Though i think variable messes up final output in docs.openshift.com